client-app/ui-kit/components/molecules/address-form/vc-address-form.vue

```diff
@@ -52,7 +52,7 @@
       <!-- Divider -->
       <div
         v-if="withPersonalInfo"
-        class="-mx-96 mb-6 mt-8 border-t border-[color:var(--color-primary)] md:mx-9 md:mb-4 md:mt-6 md:border-l"
+        class="-mx-96 mb-6 mt-8 border-t border-primary md:mx-9 md:mb-4 md:mt-6 md:border-l"
```

Comment:
to BEM

Reply:
Weird component for ui-kit, I'd like to move this component somewhere to shared components. Need to discuss and create a new task